packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -39,13 +31,16 @@ export abstract class Metric<T extends BaseBoundInstrument>
 
   constructor(
     private readonly _name: string,
-    private readonly _options: MetricOptions,
+    private readonly _options: api.MetricOptions,
```

Comment:
IIRC we have intentionally used `MetricOptions` instead of `api.MetricOptions` in order to get ride of Non-Null Assertion Operator!. This way no need to worry about missing values in the SDK and no need to assign default values the way you did here: https://github.com/open-telemetry/opentelemetry-js/pull/1137/files#diff-c46db629f283d0e2d7d273379302ec6fR94. Does it make sense?

Related comment: #415 (comment)

Reply:
hmm ok but then the api.MetricOptions was used anyway as an interface in our sdk so this is imho duplication. Also our sdk needs to be compatible with api so our sdk needs to use the api.MetricOptions anyway and you have to sync it manually. With regards to missing values maybe I'm missing something but if we expect our sdk to work with our api we should create a default values anyway not matter if we use a separate options interface or not as our sdk should work with anything (any 3rd party) that is api compatible and vice versa. I might revert it but I really think this is duplication and we endup with casting the interface once as api and once as our sdk and still we will have problem I mentioned.

Reply:
I largely agree with your comment. This kind of duplication is adding more confusion than initially intended. Also, it is hard to keep them in sync for longer runs.